packages/neos-ui/src/Containers/Modals/NodeCreationDialog/index.js

```diff
@@ -155,6 +155,16 @@ export default class NodeCreationDialog extends PureComponent {
         );
     }
 
+    editorNeedsAdditionalSpace(elementConfiguration) {
+        const editorPaths = [
```

Comment:
Hum, not sure if this is a good idea with a static list of editors.
What if I have my own editor based on one of them or at least on the selectbox?

Isn't it possible to change the styling to check for our selectbox and if it's the last element add some margin?
Or improve the editors css to adapt to the situation?

Reply:
Maybe #2735 can be fixed too, if we find a more general solution.

Reply:
Hi, thanks for the feedback! It's been a while, but I think the problem was that the editor is not aware where it is rendered (or at what position). I'm not too happy with the solution either, but it's the most efficient way I got rid of this annoying editor problem. Or find a way how the select options can overflow the dialog, so nothing is cut off, but that's not a simple task either and brings its own set of issues with it 🙈